packages/website/README.md

```diff
@@ -22,21 +22,26 @@ and Next.js will create a page for each of them that has a `slug` parameter in t
 The content that is not related to components like our "Getting started" guide is located in the `/content` folder here in the website package.
 To simplify the maintainability of this content, we ask you to keep the files in the same structure as if they were routes in Next.js style
 where the slug of the page defines its location in the folders.
-For example, if you have a page with the slug `/introduction/getting-started`, the structure will look like:
+if you have a page with the slug `/tokens/colors`, the structure will looke like:
```

Comment:
What happened here? 🤔

Reply:
Then there are some typos:
```suggestion
For example, if you have a page with the slug `/tokens/colors`, the structure will look like:
```
or
```suggestion
If you have a page with the slug `/tokens/colors`, the structure will look like:
```

Reply:
@gui-santos I don't want to block this PR, you can merge as it is and then follow up